### PR TITLE
EFS support

### PR DESCRIPTION
**Pull request by michaelwittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by michaelwittig** [comment]
test

**Comment by michaelwittig** [comment]
wait for cfn-lint update

**Comment by michaelwittig** [comment]
test

**Comment by michaelwittig** [comment]
test

File: module.yml [hunk 1]
```diff
@@ -456,6 +473,8 @@ Resources:
           - !If [HasSidecarEnvironment1Secret, {Name: !Ref SidecarEnvironment1Key, ValueFrom: {'Fn::ImportValue': !Sub '${SidecarEnvironment1SecretModule}-Arn'}}, !Ref 'AWS::NoValue']
           - !If [HasSidecarEnvironment2Secret, {Name: !Ref SidecarEnvironment2Key, ValueFrom: {'Fn::ImportValue': !Sub '${SidecarEnvironment2SecretModule}-Arn'}}, !Ref 'AWS::NoValue']
           - !If [HasSidecarEnvironment3Secret, {Name: !Ref SidecarEnvironment3Key, ValueFrom: {'Fn::ImportValue': !Sub '${SidecarEnvironment3SecretModule}-Arn'}}, !Ref 'AWS::NoValue']
+          MountPoints:
```
Comment (andreaswittig) [inline module.yml:476]: So you assume that it is possible to configure the path that the application reads/writes files?

**Review by andreaswittig — APPROVED** [review]

File: module.yml (hunk 1 above)
Comment (michaelwittig) [reply module.yml:476]: yes. the same approach that we take inside the ec2 modules. we could make it configurable?

Comment (andreaswittig) [reply module.yml:476]: 👍

**Review by andreaswittig — APPROVED** [review]

**Pull request merged**